src/ModelContextProtocol/Server/McpServerToolCreateOptions.cs

```diff
@@ -132,6 +133,14 @@ public sealed class McpServerToolCreateOptions
     /// </remarks>
     public JsonSerializerOptions? SerializerOptions { get; set; }
 
+    /// <summary>
+    /// Gets or sets the JSON schema options when creating <see cref="AIFunction"/> from a method.
+    /// </summary>
+    /// <remarks>
+    /// Defaults to <see cref="AIJsonSchemaCreateOptions.Default"/> if left unspecified.
+    /// </remarks>
+    public AIJsonSchemaCreateOptions? SchemaCreateOptions { get; set; }
```

Comment:
The related `McpServerPromptCreateOptions` would similarly require a schema options. Could you add it there as well please?